.prettierrc.js

```diff
@@ -1,8 +1,9 @@
 module.exports = {
   endOfLine: 'lf',
-  semi: true,
-  singleQuote: true,
+  semi: false,
+  singleQuote: false,
+  quoteProps: 'preserve',
   tabWidth: 2,
   trailingComma: 'es5',
-  printWidth: 100,
+  printWidth: 120,
```

Comment:
Please don't change these values they are set to match the config we use in the Strapi core repo